### PR TITLE
Closing selected timeseries not firing onselectionchange

### PR DESCRIPTION
**Pull request by follan**
#140

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-141.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-141.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesSearch/TimeseriesSearch.tsx [hunk 1]
```diff
@@ -210,19 +205,46 @@ export class TimeseriesSearch extends React.Component<
...
     ) !== -1;
 
-  onItemClose = (items: Item[]) => {
-    this.setState({ selectedTimeseries: items });
+  onItemClose = (item: Item) => {
+    const { selectedTimeseries } = this.state;
+    const closedTimeseries = selectedTimeseries.find(
+      timeseries => timeseries.id === item.id
+    );
+    console.log(closedTimeseries);
```
Comment (f1cognite) [inline src/components/TimeseriesSearch/TimeseriesSearch.tsx]: Remove

Comment (follan) [reply src/components/TimeseriesSearch/TimeseriesSearch.tsx]: Done

**Pull request merged**